### PR TITLE
Big June 2021 PR

### PR DESCRIPTION
**Pull request by AndreaCensi**
Mother issue: https://ethidsc.atlassian.net/browse/DTSW-990

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -46,8 +47,12 @@ COPY ./dependencies-apt.txt "${REPO_PATH}/"
 RUN dt-apt-install "${REPO_PATH}/dependencies-apt.txt"
 
 # install python dependencies
+ARG PIP_INDEX_URL="https://pypi.org/simple"
+ENV PIP_INDEX_URL=${PIP_INDEX_URL}
+RUN echo PIP_INDEX_URL=${PIP_INDEX_URL}
```
Comment (afdaniele) [inline Dockerfile:52]: @AndreaCensi remove it?

File: assets/environment.sh [hunk 2]
```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 
+set -eu
+export DEBUG=1
```
Comment (afdaniele) [inline assets/environment.sh:4]: @AndreaCensi this should be `0` by default

File: assets/environment.sh [hunk 3]
```diff
@@ -51,3 +56,20 @@ dt-exec() {
...
+
+dt-exec-FG() {
+    cmd="$@"
+    eval "stdbuf -o L ${cmd%&} 1>&2 "
+}
+
+copy-ros-logs() {
+
+  find /tmp/log  -type f  -name "*.log" -exec cp {} /challenges/challenge-solution-output \;
+
```
Comment (afdaniele) [inline assets/environment.sh:74]: @AndreaCensi remove empty (surrounding) lines?

**Review by afdaniele — CHANGES_REQUESTED** [review]

File: assets/environment.sh (hunk 2 above)
Comment (AndreaCensi) [reply assets/environment.sh:4]: How can debug be set?

File: assets/environment.sh (hunk 3 above)
Comment (AndreaCensi) [reply assets/environment.sh:74]: sure

**Pull request merged**